web.js

```diff
@@ -1,5 +1,7 @@
 var express = require('express');
 var url = require('url');
+var nodemailer = require('nodemailer');
+var transporter = nodemailer.createTransport();
 
```

Comment:
You need some type of configuration for this to work.  SMTP or Gmail or something.